moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -50,7 +50,7 @@
 using namespace std::chrono_literals;  // for s, ms, etc.
 
 static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_servo.servo_calcs");
-constexpr auto ROS_LOG_THROTTLE_PERIOD = std::chrono::nanoseconds(30ms).count();
+constexpr size_t ROS_LOG_THROTTLE_PERIOD = 3000;  // milliseconds
```

Comment:
I just think `3000; // milliseconds` is more readable than `std::chrono::...whatever`. I would have to google how to use std::chrono properly.

Will change if you feel strongly though.

Reply:
How about `3000ms` (https://en.cppreference.com/w/cpp/chrono/operator%22%22ms)?

Reply:
In general, I prefer expressiveness through types and not comments (though I like comments).
The issue becomes, it's immediately cast to a `size_t` which definitely doesn't enforce milliseconds.
I would prefer the use of `chrono` but won't be a stickler for it here because that type safety should really be propagated to point of use (which I guess is a `ros::Duration, so that should really be the type here.

@henningkayser he'd have to do `constexpr size_t ROS_LOG_THROTTLE_PERIOD = (3000ms).count();`

Reply:
`ROS_LOG_THROTTLE_PERIOD` doesn't need to be a `size_t`, it could be an `rclcpp::Duration` that returns `nanoseconds()` to the logging call. I find it a little odd that the logging macro doesn't support the actual duration type.

Reply:
Absolutely, though I assumed increasing the magnitude of the change would have been out of scope for this PR. My guess is that `ROS_INFO` just wraps `printf` and so has a limited type set that it can support. `ROS_INFO_STREAM` uses the insertion operator so could print more interesting things.

Reply:
I changed it to `std::chrono::milliseconds(3000).count();`